### PR TITLE
Facid no longer usable from normal spray bottles

### PR DESCRIPTION
**Pull request by Sometinyprick**
Okay I changed it around a bit and now spraying facid from a sprayer will cause it to melt and damage you.
It's not for sacid because it's not really that deadly as it only does around 35 brute damage as opposed to the insta crit facid does.
The damage from spraying facid is 50 brute and it deletes the spray bottle.
As a note unlike adding a check if the reagent container has facid you can't just stick water or another reagent in with the facid to bypass the check.
Also I swear I properly tested it this time.
(and yes I know there is that stupid commit but it's better than the original 11 commits)

### CODE REVIEW

**Comment by Steelpoint** [comment]
Not to say I'm for or against this but why would it only melt the spray bottle if the contents are being ejected? Not when they are being inserted.

**Comment by Sometinyprick** [comment]
It's supposed to be melting the plastic bit at the top, I always imagined you removed that when you put chemicals in.  However I have no idea what the actual name of the plastic bit is called and saying "The plastic bit at the top melts making the spray bottle unusable!" sounds a little silly. So I just left it as the "The spray bottle melts!".

**Comment by vista-** [comment]
Nozzle? Pump?

**Comment by Sometinyprick** [comment]
I thought about nozzle and I realized when you change spray mode it says "You remove the nozzle!" so that wouldn't make much sense. Pump sounds alright though thanks.

**Comment by Dorsisdwarf** [comment]
Oh yeah - you should change the spray mode thing to "You adjust the nozzle", not "You remove the nozzle"

**Comment by Sometinyprick** [comment]
Oh well I already changed it, instead of just adding a new commit I'd rather keep it as is.

File: code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -43,6 +43,14 @@
 		user << "<span class='notice'>\The [src] is empty!</span>"
 		return
 
+	if(!facidallowed)
+		if(reagents.has_reagent("facid"))
+			qdel(src)
+			var/mob/living/carbon/human/M = user
+			M.adjustBruteLoss(50) //This is high because actually spraying yourself with facid will do around 100 brute, this is supposed to represent a fraction of acid.
+			M.visible_message("<span class='danger'>[user] sprays acid from his [src]! It melts the pump covering him in acid!</span>", \
+			"<span class='userdanger'>The pump of [src] melts as you spray acid from it making it unusuable! and covering you in acid!</span>")
```
Comment (PKPenguin321) [inline code/modules/reagents/reagent_containers/spray.dm]: >The pump of [src] melts as you spray acid from it making it unusuable! and covering you in acid!

Fix the spelling of unusable and get that exclamation point off of it. Maybe put a comma just before "making" so the sentence sounds more fluid.

**Comment by RemieRichards** [comment]
You should call the acid_act() of the user, literally splashing them in the acid.

**Comment by Sometinyprick** [comment]
I thought about doing that but It's a bit too harsh of a punishment. 
I mean it will literally kill you and melt most of your stuff, assuming your a chemist.

**Comment by Fox-McCloud** [comment]
Personally, I'd just have it melt the bottle when you try to use sulfuric or fluorosulfuric, instead of damaging the user. It's not intuitive that it would damage you and this is only going to cause frustration on player's behalf; with the bottle melting (delete+spawn cleanable gooey mass), it gets the point across just as effectively without it being a "screw you!" kinda deal.

**Comment by Sometinyprick** [comment]
@Fox-McCloud Yea it is a little punishing, I'd rather keep the damage though but I'll half it. As regards to giving it a new object it caused a bunch of problems and runtimes so I left that out.  The easy way to do it would be to call acid_act() but that would cause  a lot more damage and melt a chunk of your shit.

**Comment by paprka** [comment]
I don't like the idea of being able to use this at all, just have it melt the bottle when you put the acid in, it's a plastic spray bottle.

**Comment by Sometinyprick** [comment]
I disagree, I believe acid can be stored in plastic bottles.
Not like it matters because it doesn't actually spray anything, it just melts.

**Comment by paprka** [comment]
>can put acid in plastic bottles
>can't spray it through the plastic nozzle or it melts

i r8 it 8/8

**Comment by Fox-McCloud** [comment]
Technically speaking, the acids shouldn't react and melt plastics (since acids are usually used in the production of plastics)...but then again, the spraybottles are probably super cheap plastic too, so....who knows.

Personally, I think it would be better to just nerf acids rather than add in "can't use this" mechanics.

I know Goon handled it by having minimum volume to impact someone--it's a high enough level that dumping it on someone or using a smoke is effective, but spraying someone with it is not (sulfuric has no minium volume, fluorosulfuric does). Some system similar to this would also largely solve the problem without any snowflake/"gamey" checks.

**Comment by Dorsisdwarf** [comment]
What, you mean a special check to make sure you're hitting them with a certain amount of a certain chem instead of just making it melt the thingy is less snowflakey (and less retarded)?

"Hey look I can spray this guy with sulfuric acid and melt him, but the more powerful harder to get acid just bounced right off! Time and effort well-spent!"

**Comment by phil235** [comment]
Is that PR still worked on?

**Comment by Sometinyprick** [comment]
Totally forgot about this
no idea how to easily view the merge conflicts, i also lack the effort to learn how to because I'm busy with a bunch of irl shit at the moment

**Pull request closed without merging**